offgridplanner/steps/views.py

```diff
@@ -1,8 +1,6 @@
 import os
 
-import pandas as pd
```

Comment:
if you remove unused lines, better make this in a single commit  for all unused lines with a message like "Remove unused imports"